muzzle/src/test/groovy/io/opentelemetry/javaagent/tooling/muzzle/ReferenceMatcherTest.groovy

```diff
@@ -209,7 +212,7 @@ class ReferenceMatcherTest extends Specification {
   def "should fail #desc helper classes that does not implement all abstract methods"() {
     given:
     def reference = ClassRef.builder(className)
-      .setSuperClassName(TestHelperClasses.HelperSuperClass.name)
+      .setSuperClassName(TestAbstractSuperClass.name)
```

Comment:
This was needed because of the change in `HelperReferenceWrapper` that added `!helperClassPredicate.isHelperClass(className)` check. Now classes that are in `io.opentelemetry.instrumentation` are matched by that check which made a couple of tests fail. I believe the intent of the test was to test injected helper extending a class from application. To ensure that the class that the test extends is treated as an application class and not as a helper I had to copy it into a different package. I also tried moving `TestHelperClasses` but that made a bunch of other tests fail.